pkgs/development/python-modules/pluggy/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildPythonPackage, fetchPypi, pytest, six, doCheck ? true }:
```

Comment:
doCheck should not be there. Its not used, and overriding that kind of attribute should be done with `overridePythonAttrs`.

Reply:
Okay, good to know. It was a left over from the hypothesis module.